packages/sdk-js/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
 	"compilerOptions": {
 		"target": "es5",
-		"lib": ["es5", "DOM"],
```

Comment:
Didn't you mention _adding_ DOM in that other PR? 🤔

Reply:
Yes,  #3479. That's why I removed dom, I was hoping that @nickrum will add it. I'll add another commit with added dom

Reply:
I added dom in last commit

Reply:
Sorry, I was a little busy today.